src/hacking/debugging-on-openharmony.md

```diff
@@ -0,0 +1,88 @@
+# Debugging on OpenHarmony
+
+It's recommended to read the general advice from the [Debugging section](debugging.md) first.
+This section will only cover the differences that need to be considered on OpenHarmony
+
+## Starting servoshell from the commandline
+
+When debugging it can often be useful to start servo from the commandline instead of clicking the app icon,
+since we can pass parameters from the commandline.
```

Comment:
It looks like the book is written with one sentence per line, so please do that here as well. This doesn't hold for code blocks though, as far as I understand.